File: GARI_Umbria_2026-08-03/DotNet/AgronicaCoreStd/AgronicaCoreModelsSTD/anagrafiche/CatastoCentroAziendale.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace AgronicaCoreModelsSTD.anagrafiche
{
    public class CatastoCentroAziendale
    {
        public CentroAziendale.PK centro { get; set; }
        public ParticelleCatastali particella { get; set; }
        public List<PossessoParticella> possessiParticella { get; set; }
        public bool flag_cancellazione { get; set; }
        public CatastoCentroAziendale()
        {
            flag_cancellazione = false;
        }
    }
}

File: GARI_Umbria_2026-08-03/GiasNG/src/app/Model/anagrafiche/CatastoCentroAziendale.ts
import { PKCentroAziendale } from './CentroAziendale';
import { ParticelleCatastali } from './ParticelleCatastali';
import { PossessoParticella } from './PossessoParticella';

export class CatastoCentroAziendale {
    centro: PKCentroAziendale;
    particella: ParticelleCatastali;
    possessiParticella: PossessoParticella[];
    flag_cancellazione: boolean;

    constructor() {
        this.flag_cancellazione = false;
    }
}
